web/src/views/building/BuildingScreen.vue

```diff
@@ -6,6 +6,11 @@
         id="banner"
         src="https://unsplash.com/photos/95YCW2X5jUc/download?force=true&w=1920"
       />
+      <RemovedCard
+        :show="building.deleted"
```

Comment:
Ik zou bij alle v-shows nog checks doen of we admin zijn. Dit omdat een superstudent/user misschien per ongeluk ook op deze pagina's zou kunnen belanden

Reply:
Ik voeg het toe